### PR TITLE
[bug 1422899] Remove old firefox/new template

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Removes old `/firefox/new/` template.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1422899

## Testing
- Probably a good idea to run `gulp static:clean` prior to testing this PR.
- Check I didn't remove any other assets that may still be needed?
- Check translations still appear as expected.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/new/scene2.html [hunk 1]
```diff
@@ -2,22 +2,17 @@
...
+{% add_lang_files "firefox/new/quantum" %}
 
 {% extends "firefox/new/base.html" %}
 
 {% block extra_meta %}
   <meta name="robots" content="noindex,follow">
 {% endblock %}
 
-{% block optimizely %}
-  {% if switch('firefox-new-scene2-optimizely', ['en-US']) %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/new/scene2.html]: Note to self: I need to rebase this branch again so this gets added back in.

**Comment by alexgibson** [comment]
@jpetto @craigcook do either of you have some free time for a review? This isn't urgent so-to-speak, but it is incurring a lot of manual rebasing to keep the branch up to date.

File: bedrock/firefox/templates/firefox/includes/schemaorg-app.html [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-  <meta itemprop="url" content="{{ settings.CANONICAL_URL }}/firefox/new/">
-  <meta itemprop="image" content="{{ self.product_logo() }}">
-  <meta itemprop="logo" content="{{ self.product_logo() }}">
-  <div itemprop="manufacturer" itemscope itemtype="http://schema.org/Organization">
-    <meta itemprop="name" content="{{_('Mozilla')}}">
-  </div>
-  <div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-    <meta itemprop="price" content="0">
-  </div>
-</div>
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/includes/schemaorg-app.html]: We're good dropping this file? Is it no longer useful?

File: bedrock/firefox/templates/firefox/new/scene1.html [hunk 3]
```diff
@@ -2,201 +2,135 @@
...
-  <div class="foreground">
-    <aside id="newsletter-subscribe">
-      <div class="container">
-        {{ email_newsletter_form(title=_('Keep up with<br> all things Firefox.'), button_class='button-light', spinner_color='#fff') }}
+</div>
+{% endblock %}
+
+{% block main_content %}
+<header>
+{% if l10n_has_tag('firefox_quantum_new_headline') %}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/new/scene1.html]: I think we might be able to drop this conditional? Looks like coverage is good for [both](https://l10n.mozilla-community.org/langchecker/?website=0&file=firefox/new/quantum.lang&action=translate&show#string33) [strings](https://l10n.mozilla-community.org/langchecker/?website=0&file=firefox/new/quantum.lang&action=translate&show#string34).

File: bedrock/firefox/templates/firefox/includes/schemaorg-app.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/includes/schemaorg-app.html]: I pondered this also. My personal view on schema.org is that it isn't really that useful (happy to be proven wrong with some measurable results). That aside though, the current quantum template is missing the strings used here, so we can't just bolt it back on. I notice there is a new task in Taiga to add metadata for the Firefox logo etc, so perhaps we can revisit this then. Thoughts?

File: bedrock/firefox/templates/firefox/new/scene1.html (hunk 3 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/new/scene1.html]: Good call 👍

**Comment by alexgibson** [comment]
@jpetto thanks for the review, updated.

**Comment by flodolo** [comment]
Note: need to remove:
* tag `firefox_quantum_new_headline`
* strings `Meet Firefox Quantum` and `New. Fast. For good.`

File: bedrock/firefox/templates/firefox/includes/schemaorg-app.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/includes/schemaorg-app.html]: Hmm, perhaps I could ressurect this file using some of the existing strings in the new template. Let me have a look 👍

**Comment by alexgibson** [comment]
@jpetto I added another commit with the schema.org bits. Let me know what you think.

**Comment by jpetto** [comment]
> @jpetto I added another commit with the schema.org bits. Let me know what you think.

Going to be honest, I also don't know the value of the schema.org content, but I can't see any issues with the update here. I'm in favor of keeping it around for now, as it doesn't appear to *hurt* anything.

Squash time?

**Comment by alexgibson** [comment]
Squashed!

**Pull request merged**